Source/DafnyCore/AST/Grammar/ProgramParser.cs

```diff
@@ -35,7 +35,7 @@ public class ProgramParser {
     CancellationToken cancellationToken) {
     var options = errorReporter.Options;
     var builtIns = new SystemModuleManager(options);
-    var defaultModule = new DefaultModuleDefinition(files.Where(f => !f.IsPreverified).Select(f => f.Uri).ToList());
```

Comment:
Seems like the field RootSourceUris can be removed from `DefaultModuleDefinition` entirely, since it's only access on line 42 in this method.